UPGRADE-3.0.md

```diff
@@ -1,6 +0,0 @@
-UPGRADE FROM 2.X to 3.0
-=======================
-
-Nothing was documented since now.
-
-If you find BC break and want to contribute, feel free.
```

Comment:
https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1109 should have come with an upgrade note, let's add one now?

Reply:
I can, I'll add in 3.x.

I removed the 3.0, since it's pretty old and doesn't add value.

Reply:
You can do it right here since you are targeting 3.x

Reply:
Oh right you meant UPGRADE-3.x.md, got it!